query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/opensearch/persist/UpdateOpenSearchConfigurationSpec.groovy

```diff
@@ -160,6 +158,38 @@ class UpdateOpenSearchConfigurationTest extends Specification {
         report.messages().get(0).code == DefaultMessages.FAILED_PERSIST
     }
 
+    def 'Updating with flag password sends service properties without password'() {
+        setup:
+        updateOpenSearchConfiguration.setValue(createUpdateFunctionArgs(FLAG_PASSWORD))
+        serviceActions.read(_ as String) >> [(ID): TEST_SOURCENAME]
+        serviceReader.getServices(_, _) >> []
+        configurator.commit(_, _) >> mockReport(false)
+
+        def expectedUpdateConfig = getUpdateOpenSearchProps(null)
```

Comment:
I did mean to change this to `FLAG_PASSWORD` since it has the same effect as passing in `null` due to the way I am making this expected update config map.